apps/web/src/components/media-gallery/media-gallery.tsx

```diff
@@ -76,7 +76,7 @@ export default function MediaGallery({ media }: MediaGalleryProps) {
               alt={t('common:statuses.Selected Image')}
               className={css.contain}
               layout="fill"
-              src={currentMedia}
+              src={`${currentMedia}?width=700&quality=95`}
```

Comment:
Simon already set a `sizes` prop on this one so there's no need for any changes here.